test/xapi/problem_test.rb

```diff
@@ -67,8 +67,8 @@ def test_naming
   end
 
   def test_test_file
-    expected_test_file = "assert 'one'\n"
+    expected_test_hash = { "one_test.tst" => "assert 'one'\n" }
```

Comment:
We can call this `expected` rather than adding details about the data type to the variable name.